iface/i3blocks.conf

```diff
@@ -5,3 +5,5 @@ command=$SCRIPT_DIR/iface
 #ADDRESS_FAMILY=inet6?
 color=#00FF00
 interval=10
+# set this to 1 to display the name of the connected WIFI interface instead of the IP address.
+display_wifi_name=0
```

Comment:
a bit atypical perhaps. I made this variable all lower-case. Throughout the blocks upper-case and lower-case is mixed, that's a bit confusing. blocks-settings are lower-case so I thought I make this lower-case as well.